theme_buzzy/views/images_library.xml

```diff
@@ -236,5 +236,77 @@
     <field name="name">website.s_showcase_default_image</field>
     <field name="url">/theme_buzzy/static/src/img/photos/s_showcase.jpg</field>
 </record>
```

Comment:
I changed the images and removed the current ones since they weren't used anymore. (https://github.com/odoo/design-themes/compare/a06b0684e1d01eb9b7f2c8ffe1ee7ff57723d19d..dafc021262ed4c15eba9df3f238c1e660cf2a0c7)
<img width="1408" alt="Capture d’écran 2024-09-28 à 17 10 54" src="https://github.com/user-attachments/assets/58738632-333c-4179-92b3-93358859466b">